octoprint_mrbeam/services/document_service.py

```diff
@@ -47,7 +47,8 @@ def get_document_simple_for(self, definition, language):
     def _get_title_translated(self, definition):
         title_key = string_util.separate_camelcase_words(definition.mrbeamdoc_type.value)
         title_translated = gettext(title_key)
-        self._log_error_on_missing_translation(title_key, str(title_translated))
+        if title_translated:
```

Comment:
Would it make sense to log a warning if there's no translation? Or is this a common/valid case?